srcpkgs/cgdb/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'cgdb'
 pkgname=cgdb
-version=0.7.1
-revision=2
+version=0.8.0
+revision=1
 build_style=gnu-configure
 configure_args="ac_cv_rl_version=6.3
  ac_cv_file__proc_self_status=yes"
 hostmakedepends="flex texinfo"
 makedepends="ncurses-devel readline-devel"
 depends="gdb"
 short_desc="Curses-based interface to the GNU Debugger"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="meator <meator.dev@gmail.com>"
 license="GPL-2.0-only"
 homepage="http://cgdb.sourceforge.net"
```

Comment:
this link doesn't work

Reply:
I do not understand you. Could you explain that?

Reply:
This link redirects me to https://cgdb.github.com/ which returns a github 404 page:
![Screenshot from 2022-01-20 18-29-15](https://user-images.githubusercontent.com/23639164/150390863-407c0853-ed92-4202-aafc-ad02a253c128.png)

https://cgdb.github.io/ works as expected